### PR TITLE
Improved app page to conform more closely with other FOSSASIA projects. 

### LINKED ISSUE #108: Improve app page
Please change the app page to conform more closely with other FOSSASIA projects, e.g. Badgeyay
* Add a top area with a logo on the left and a dropdown menu of items on the right
* Use the text here as a logo http://patorjk.com/software/taag/#p=display&f=Flower%20Power&t=meilix
* Take out the large placeholder logo
* Change "Event Name" to "Name of Your Distribution"
* Change "Event Link" to "Browser Home Page"
* Change "A webapp which can generate iso for you" to "Create Your own lubuntu Distribution"
* Add a link to the current deployment on the bottom

Compare to Badgeyay

screenshot from 2017-12-18 10-58-02

### PR DESCRIPTION
**Pull request by rish4bhn**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Changes to the app page to conform more closely with other FOSSASIA projects, e.g. Badgeyay.
As the **navbar** was redesigned according to the **new** requirements, a lot of code for the older menu has been _removed_ (100+ lines of JS, CSS animations).

**Screenshot**

meilix_final

#### Changes proposed in this pull request:

- [x] Added a **navbar** with a logo on the left and a **dropdown menu** of items on the right.
- [x] Used the text here as a logo http://patorjk.com/software/taag/#p=display&f=Flower%20Power&t=meilix
- [x] Took out the large placeholder logo
- [x] Added a footer identical to the BadgeYAY footer 
- [x] Added a link to the **current deployment** and **Releases** page to the footer
- [x] Removed unnecessary JS, CSS

Fixes #108

### CODE REVIEW

**Comment by meets2tarun** [comment]
Please deploy the webapp on Heroku and give the link here

**Comment by abishekvashok** [comment]
@rish4bhn pls see the dropdown in badgeyay.

**Comment by rish4bhn** [comment]
@meets2tarun I have no experience deploying flask apps, could you please clone my repo and check it out: https://github.com/rish4bhn/meilix-generator

@abishekvashok Oh, you wanted the exact same dropdown? I can do it but not in the next couple days(_travelling right now_), so can we add that in a future PR?

Thanks.

**Comment by rish4bhn** [comment]
@meets2tarun Did you check it out?

**Comment by abishekvashok** [comment]
@rish4bhn why should we settle for a new or, when you can do it here :). We can wait, till you get back :)

File: templates/index.html [hunk 1]
```diff
@@ -1,82 +1,93 @@
 <!doctype html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" type="text/css" href="{{ url_for('static',filename='css/bootstrap.min.css') }}">
   <link rel="stylesheet" href="{{ url_for('static',filename='css/menu.css') }}">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (tabesin) [inline templates/index.html]: Why external stylesheet?

File: templates/index.html [hunk 2]
```diff
@@ -1,82 +1,93 @@
...
       </div>
     </div>
-  </div>
-</form>
+  </form>
 
-<script type="text/javascript" src="{{ url_for('static', filename='js/menu.js') }}"></script>
+  <footer class="footer">
+    <div class="container text-center">
+      <span class="text-muted"> Grab the latest pre-built ISO
```
Comment (tabesin) [inline templates/index.html]: Why should an app user want to do this?

File: .gitignore [hunk 3]
```diff
@@ -106,3 +106,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+Visual Studio Code config
```
Comment (tabesin) [inline .gitignore]: Why is that needed?

**Comment by rish4bhn** [comment]
@tabesin, the logo and the other features were requested in https://github.com/fossasia/meilix-generator/issues/108.
Cool.

**Comment by mariobehling** [comment]
@rish4bhn Thank. Pity you closed this. I think it was a good work in progress PR. Let me see if I can merge this and we can progress from here.

File: .gitignore (hunk 3 above)
Comment (mariobehling) [reply .gitignore]: Please take this out.

File: templates/index.html (hunk 1 above)
Comment (mariobehling) [reply templates/index.html]: It would be better without external stylesheet.

**Comment by mariobehling** [comment]
I am merging this as work in progress. Please help to resolve the dependencies and small issues left here. Cheers. Good job!
